altair/examples/correlation_matrix.py

```diff
@@ -0,0 +1,52 @@
+"""
+Correlation matrix
+--------------
+This example shows how to create a correlation matrix and a heatmap from the iris dataset.
+"""
+# category: other charts
+# Load packages
+
+import altair as alt
+from vega_datasets import data
+
+# Load the data
+df_iris = data.iris()
+
+# Create a correlation matrix
+corrMatrix_line = df_iris.corr().round(2).reset_index().rename(columns = {'index':'Var1'}).melt(id_vars = ['Var1'],
+                                                                                                value_name = 'Correlation',
+                                                                                                var_name = 'Var2')
+# Create the heatmap first
+heatmap = alt.Chart(corrMatrix_line).encode(
+    alt.Y('Var1:N', title = ''),
+    alt.X('Var2:N', title = '', axis=alt.Axis(labelAngle=20))
+).mark_rect().encode(
```

Comment:
I would move `mark_rect()` to directly after `alt.Chart()` and only have a single call to `encode()` like many of the other examples.